stripper/ze_ancient_wrath_v2_test27_m5.cfg

```diff
@@ -0,0 +1,203 @@
+;add timelimit to stage 6 laser boss fight
+add:
+{
+	"classname" "logic_relay"
+	"targetname" "antidelay_lasers"
+	"origin" "100 100 100"
+	"spawnflags" "0"
+	"StartDisabled" "0"
+	"OnTrigger" "commandCommandsay || YOU TOOK TOO LONG ||40-1"
+	"OnTrigger" "playerRunScriptCodeforeach(k,_ in{SetHealth=0}){if(self.GetTeam()==3&&self.GetHealth()>0)EntFireByHandle(self, k,(0).tostring(),0,null,null)}41-1"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "path_track"
+		"targetname" "Train_path_extreme4"
+	}
+	insert:
+	{
+		"OnPass" "antidelay_lasers,Trigger,,10,1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_physbox"
+		"targetname" "boss_quad_encounter_01_hitbox"
+	}
+	insert:
+	{
+		"OnUser1" "antidelay_lasers,CancelPending,,0,-1"
+	}
+}
+
+;fix fuel game text breaking other game_text channels
```

Comment:
Can you add a loose .nut notice similar to [this](https://github.com/gflclan-cs-go-ze/ZE-Configs/blob/master/stripper/ze_8bit_csgo3.cfg#L2)? Would be easy to miss adding the .nut for anyone else using this config otherwise.